atlassian-terraform/sumologic.tf

```diff
@@ -64,7 +64,7 @@ resource "null_resource" "install_jira_cloud_app" {
 
   provisioner "local-exec" {
     command = <<EOT
-        curl -s --request POST '${var.sumo_api_endpoint}apps/019757ca-3b08-457c-bd15-7239f1ab66c9/install' \
+        curl -s --request POST '${var.sumo_api_endpoint}v1/apps/019757ca-3b08-457c-bd15-7239f1ab66c9/install' \
```

Comment:
Try if we use /v1 here does it still work if user has /? That way we dont really need user to keep the /  in the sumo_api_endpoint

Reply:
It didn't work, so instead, I removed slash from the endpoint before curl i.e.
  sumo_api_endpoint_fixed = replace(var.sumo_api_endpoint, "api/", "api")